### PR TITLE
Improve README formatting and fix compass command

### PR DESCRIPTION
**Pull request by adamzap**
I changed the two unordered lists to ordered lists and improved line wrapping

### CODE REVIEW

**Comment by adamzap** [comment]
I'm leaving this as two different commits because they have separate concerns and the second depends on the first.

**Comment by adamzap** [comment]
Also, I the documented `compass` command to generate the patch for #45:

`compass compile --sass-dir trac-env/htdocs/css/ --css-dir trac-env/htdocs/css/ -e development --sourcemap --force`

@bmispelon It looks like you used the `sassc` command on #40 and #42, resulting in large diffs.

Am I correct in thinking that the `compass` command is correct?

I am using #34 as a model since it was merged.

**Comment by bmispelon** [comment]
Hi @adamzap,

The main djangoproject repo moved to using libsass (a python implementation of sass) so that we wouldn't need to have ruby installed.

The large diff on the compiled CSS might be because of that but I don't think it's too much of a big deal. Ideally, the CSS files should be generated during deploy time but for ease of use they aren't currently. In the meantime, I tend to treat them as binary blobs basically so the diff doesn't really matter.

File: README.rst [hunk 1]
```diff
@@ -7,15 +7,15 @@ Local install
...
-* Install the requirements in the same virtualenv as djangoproject.com.
-* Follow the installation instructions in djangoproject/README.rst (especially
-  the database creation).
-* Use ``psql -U code.djangoproject -d code.djangoproject -c "INSERT INTO permission (username, action) VALUES ('anonymous', 'TRAC_ADMIN')"``
-  to give all permissions to the anonymous user.
-* Use the command ``tracd --port 9000 -s trac-env`` to serve Trac locally.
-* If you've modified the ``trackhack.scss`` file, use
-  ``sassc trac-env/htdocs/css/trachacks.scss trac-env/htdocs/css/trachacks.css -s compressed``
-  to compile it to CSS.
+1. Install the requirements in the same virtualenv as djangoproject.com.
```
Comment (bmispelon) [inline README.rst]: This list is more of a random collections of tips so I think it can remain an unordered list.

File: README.rst [hunk 2]
```diff
@@ -23,12 +23,11 @@ How to port the CSS from djangoproject.com
...
-    compass compile --sass-dir trac-env/htdocs/css/ --css-dir trac-env/htdocs/css/ -e development --sourcemap --force
-
+1. Copy the generated CSS:
+   ``cp ../djangoproject.com/static/css/*.css trac-env/htdocs/css/``
+2. Copy _utils.scss (needed by trackahacks.scss):
+   ``cp ../djangoproject.com/static/scss/_utils.scss trac-env/htdocs/css/``
+3. Copy the javascript directory:
+   ``cp -rT ../djangoproject.com/static/js trac-env/htdocs/js``
+4. Compile trackhacs.scss:
+   ``compass compile --sass-dir trac-env/htdocs/css/ --css-dir trac-env/htdocs/css/ -e development --sourcemap --force``
```
Comment (bmispelon) [inline README.rst]: Can you change this command to use `sassc` instead (I missed that when I added the list above).

**Comment by adamzap** [comment]
Thanks for the clarification!

I made the changes

**Comment by bmispelon** [comment]
Merged in b4c2a1e8fdc607a3c6cb70faae4cc215c779c78b.

Thanks! :rocket:

**Pull request closed without merging**